src/sprites/entities/player.py

```diff
@@ -56,18 +60,20 @@ def __init__(
         )
 
         # movement
+
         self.controls = Controls
         self.load_controls()
+        self.originalSpeed = 250
```

Comment:
Since the remainder of the codebase favours snake case (split words using underscores) instead of camel case, this name should be changed to use snake case as well for code consistency.

```suggestion
        self.original_speed = 250
```